index.html

```diff
@@ -1,314 +1,312 @@
-    <link rel="stylesheet" media="screen and (min-width: 768px)" href="style_desktop.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins:ital@0;1&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      media="screen and (max-width: 767px)"
+      href="index.css"
+    />
+    <link
+      rel="stylesheet"
+      media="screen and (min-width: 768px)"
+      href="style_desktop.css"
+    />
+    <script type="text/javascript" src="index.js"></script>
 
     <title>Portfolio</title>
-</head>
+  </head>
 
-<body>
-    <nav class='NavBar1'>
-        <a href="#" class="h2_1">Camilux</a>
-        <script type="text/javascript" src="index.js"></script>
-        <button class="nav_menu1" onclick="showNav();" id="clickMe1"><img src="./images/clipart365828.png" alt="" class="hamburger" /></button>
-        <button class="nav_menu1" onclick="hideNav();" id="clickMe2"><img src="./images/HideNavIcon.png" alt="" class="hamburger2" /></button>
-        <ul class="nav_list" id="My_navList">
-            <li class="li_nav"><a href="#section2" class="nav_menu" onclick="hideNav();">Portfolio</a></li>
-            <li class="li_nav"><a href="#section_aboutMe" class="nav_menu" onclick="hideNav();">About</a></li>
-            <li class="li_nav"><a href="#contact_form" class="nav_menu" onclick="hideNav();">Contact</a></li>
-            <li class="li_nav" id="UnionIcon">
-                <a href="#" class="nav_menu"><img src="./images_desk/Union.png" alt="" class="hamburger2" /></a>
-            </li>
-        </ul>
+  <body
+    onload="displayPrjct('Profesional Art Printing Data More', 'Ipsum has been the industrys standard dummy text ever since the 1500s when an unknown printer took a galley of type ','./images/portfolioBackground.png','1');
+                displayPrjct('Data Dashboard Healthcare', 'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industrys  ','./images/portfolioBackground.png','2');
+                displayPrjct('Website Protfolio ', 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard ','./images/portfolioBackground.png','3');
+                displayPrjct('Profesional Art Printing Data More', 'Ipsum has been the industrys standard dummy text ever since the 1500s when an unknown printer took a galley of type ','./images/portfolioBackground.png','4');
+                displayPrjct('Data Dashboard Healthcare', 'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industrys  ','./images/portfolioBackground.png','5');
+                displayPrjct('Website Protfolio ', 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard ','./images/portfolioBackground.png','6');
+                "
```

Comment:
- [ ] It is important to keep all your JavaScript code in your folders and files, and not in the HTML itself, as part of the good practices, so please remove this and move it to any of your JavaScript files.

Reply:
Hello @[Lordkaito](https://github.com/Lordkaito) ,
I dont really understand when you say my project is not dynamic because i created everything in the javascript and I have only to enter parameters. I think It was that the requirements.
I will see how to use objects but as you say is like I didn't do anything. Please can you clarify me on how my project is not dynamic so I will know well how to improve changes.Thank you!